Makefile

```diff
@@ -44,6 +44,8 @@ ifndef EXEC
 	DOCKER_RUN_FLAGS += -it
 endif
 
+ETC_LOCALTIME = $(realpath /etc/localtime)
```

Comment:
As long as you're using GNU extensions like `$(realpath ...)`, I would go ahead and make this a simply-expanded rather than recursively-expanded variable.

```
ETC_LOCALTIME := $(realpath /etc/localtime)
```

Simply-expanded variables are my default when writing Make unless I know I need the `m4`-ish recursive expansion behavior, which can loop.

See https://www.gnu.org/software/make/manual/html_node/Flavors.html .